common/src/main/scala/com/gu/emailservices/EmailService.scala

```diff
@@ -32,12 +37,32 @@ case class EmailFields(
        |        "edition": "$edition",
        |        "name": "$name",
        |        "product": "$product"
+       |        $paymentMethodJson
        |      }
        |    }
        |  },
        |  "DataExtensionName": "$dataExtensionName"
        |}
     """.stripMargin
+
+  def paymentMethodJson: String = paymentMethod match {
+    case Some(dd @ DirectDebitPaymentMethod(_, _, _, _, _, _, _, _)) => s"""
+      ,"account name": "${dd.bankTransferAccountName}",
```

Comment:
Why not keep these field names consistent with the rest of the JSON payload by using camel-case (like ContactAttributes and EmailAddress)?

Reply:
Rachel set these up in ExactTarget and I didn't want to make her rework everything she'd done. I agree that it is a bit nasty...